drracket/drracket/tool-lib.rkt

```diff
@@ -238,8 +238,8 @@ all of the names in the tools library, for use defining keybindings
   @{Registers a pair of procedures with DrRacket's online expansion machinery. 
     (See also @racket[drracket:module-language-tools:add-online-expansion-monitor].)
     
-    The first two arguments name a procedure in a module that is loaded by 
-    @racket[dynamic-require] is specially designed separate @racket[place].
+    The procedure @racket[id] from @racket[mod-path] is loaded by 
+    @racket[dynamic-require] into a specially designed separate @racket[place].
```

Comment:
Maybe something like `@tech[#:doc '(lib "scribblings/reference/reference.scrbl")]{place}` would be better than `@racket[place]`?

Reply:
```suggestion
    @racket[dynamic-require] into a specially designed separate @tech[#:doc '(lib "scribblings/reference/reference.scrbl")]{place}.
```

Reply:
```suggestion
    @racket[dynamic-require] into a specially designed separate @tech[#:doc '(lib "scribblings/reference/reference.scrbl")]{place}.
```